core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.spec.ts

```diff
@@ -131,6 +135,28 @@ describe('Contributor dashboard page', function() {
       expect(ctrl.activeTabName).toBe(changedTab);
       expect(ctrl.showLanguageSelector()).toBe(true);
     });
+
+    it('should call scrollFunction on scroll', function() {
+      var e = document.createEvent('Event');
+      var scrollSpy = spyOn(ctrl, 'scrollFunction');
+      e.initEvent('scroll', true, true);
+      window.dispatchEvent(e);
+      expect(scrollSpy).toHaveBeenCalled();
```

Comment:
Make sure to add new lines to outline the different phases of the test structure: the test setup, action, and assertion. The wiki has more info on this here: https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests. This makes it easy to scan tests quickly for understanding. Ditto below. For really simple tests where the setup, action, and assertion are each one line each, you can omit the new lines.

```suggestion
      var e = document.createEvent('Event');
      var scrollSpy = spyOn(ctrl, 'scrollFunction');
      e.initEvent('scroll', true, true);
      
      window.dispatchEvent(e);
      
      expect(scrollSpy).toHaveBeenCalled();
```

Reply:
Done